src/screens/PetRegistration/index.tsx

```diff
@@ -386,8 +362,10 @@ const PetRegistration = () => {
 			},
 		});
 		if (result.type == "error") {
+			setLoading(false)
 			console.warn(result.error);
 		} else {
+			setLoading(false)
```

Comment:
@zeroCass Consegue fazer uma navegação aqui para outra página ? To bem perdido com essa navegação do drawer

Obs: Eu não consegui marcar a função toda mas é a função 'handleSendData' dentro do componente 'PetRegistration'